### DISCUSSION TITLE
Using dynamic data in the DataGrid component

### DISCUSSION THREAD

**Discussion by blenke** [Megabit/Blazorise#5154] [Q&A]
I would like to use the DataGrid with a dynamic set of data. I want to show the results of user surveys in the datagrid , but each survey has different amount of answers (each answer should be a column). I have the data in a datatable and converted this to a List<dynamic>  . However i can't seem to use the dynamic list on the DataGrid, the TItem="dynamic" is not accepted by the DataGrid component.
Any suggestions would be welcome.

**Comment by David-Moreira** [comment]
Try something along these lines:
https://github.com/Megabit/Blazorise/issues/3278

**Comment by blenke** [comment]
Thanks that works, however since the data is not handled by the DataGrid internally it can't sort or filter. That makes it not very usefull in my case. Perhaps you could add this as a feature for future versions.
